examples/simple/lint/L-EXC3-bad.c

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <stdlib.h>
+#include <stdio.h>
+
+void * func()
+{
+    int * retval = malloc(sizeof(int));
+    *retval = 1;
+    return retval; 
```

Comment:
I would either remove the malloc statement, or otherwise add a comment in the code saying that the function is not called, so no memory leaks, both in the bad and in the good example. That's because some static analysis tools may report an error here, and we do not want them to think that we "missed" the error.  Something as simple as:

// No memory leak reported, because the function is never invoked.